app/enterprise/0.35-x/deployment/migrations.md

```diff
@@ -4,7 +4,9 @@ title: Migrating to 0.35
 
 ### Prerequisites for Migrating to 0.35
 
-* If running a version earlier than 0.34, it is necessary to [upgrade to 0.34](/enterprise/0.34-x/deployment-guide/#upgrading-to-034) first before migrating to 0.35.
+* If running a version of Kong Enterprise earlier than 0.34, it is necessary to [upgrade to 0.34](/enterprise/0.34-x/deployment-guide/#upgrading-to-034) first before migrating to 0.35.
+* If running Kong Core versions 0.14 or 0.15, you must [upgrade to Core 1.0](/1.0.x/upgrading/) before upgrading to Enterprise 0.35. You cannot upgrade to Enterprise 0.34 first.
```

Comment:
Unless we hear otherwise...

```suggestion
* If running Kong versions 0.14 or 0.15, you must [upgrade to Kong 1.0](/1.0.x/upgrading/) before upgrading to Kong Enterprise 0.35. You cannot upgrade to Kong Enterprise 0.34 first.
```